site/docs/samples/pre-steps.md

```diff
@@ -1,6 +1,6 @@
 # Tools used by the actors
 
-- The data owner would typically register the dataset in a proprietary or open source catalog.  We have provided [katalog](../reference/katalog.md) - a thin layer acting as a replacement for the data catalog for evaluation purposes. This simplifies the sample deployment.
+- The data owner would typically register the dataset in a proprietary or open source catalog. We use [OpenMetadata](https://open-metadata.org/).
 - The data owner stores credentials for accessing the dataset in kubernetes secrets.

```

Comment:
do we need it? The credentials are stored in the OMD catalog.

Reply:
Changed the wording to: "The data owner needs to provide credentials for accessing a dataset. One way to do it is by storing credentials in kubernetes secrets."